content/docs/custom-plugins/authenticated-proxy/index.md

```diff
@@ -0,0 +1,52 @@
+---
+title: Configuring Authenticated Proxy for Custom Plugins
+lastUpdated: '2023-02-03T10:27:00.0Z'
+description: How to configure a custom proxy to identify the Backstage user for you custom plugin.
+---
+
+## Introduction
+
+Roadie provides a custom proxy service that can be configured to proxy through an access token minted by our own authentication system that can be validated by your third party service.
+
+
+## Step 1. Create an authenticated proxy configuration in Roadie
+
+You can navigate to https://your-company.roadie.so/administration/settings/authenticatedProxy. Click add item to add a new proxy.
+
+![authenticatedProxyConfig.png](authenticatedProxyConfig.png)
+
+Enter a Path and a target API as follows:
```

Comment:
We can give the user more guidance about what to enter here. I've created a few proxies recently and it's super confusing. What should they put into the path and target API fields? I know we can't tell them exactly, but if we explain what they are for they will get the idea.

Reply:
I added examples.